tools/integration_tests/util/operations/operations.go

```diff
@@ -39,6 +39,19 @@ func executeToolCommandf(tool string, format string, args ...any) ([]byte, error
 	cmdArgs := tool + " " + fmt.Sprintf(format, args...)
 	cmd := exec.Command("/bin/bash", "-c", cmdArgs)
 
+	return runCommand(cmd)
+}
+
+// Executes any given tool (e.g. gsutil/gcloud) with given args in specified directory.
+func ExecuteToolCommandfInDIrectory(dirPath, tool, format string, args ...any) ([]byte, error) {
```

Comment:
```suggestion
func ExecuteToolCommandfInDirectory(dirPath, tool, format string, args ...any) ([]byte, error) {
```
nit: Typo or intended? 😌

Reply:
Typo, thanks for pointing out! 👀